django_netjsongraph/base/admin.py

```diff
@@ -23,6 +23,20 @@ def __init__(self, *args, **kwargs):
 class BaseAdmin(TimeStampedEditableAdmin):
     save_on_top = True
 
+    def get_extra_context(self, pk=None):
+        ctx = {}
+        submit_line = 'admin/{0}/submit_line.html'.format(self.opts.app_label)
+        ctx.update({'submit_line': submit_line})
+        return ctx
+
+    def add_view(self, request, form_url='', extra_context=None):
+        extra_context = self.get_extra_context()
+        return super(BaseAdmin, self).add_view(request, form_url, extra_context)
+
+    def change_view(self, request, pk, form_url='', extra_context=None):
+        extra_context = self.get_extra_context(pk)
+        return super(BaseAdmin, self).change_view(request, pk, form_url, extra_context)
+
```

Comment:
We want the topology graph to be displayed only in the topology admin page unlike django-netjsonconfig. I just looked at the code of django-netjsonconfig and found that all the other modelAdmin inherit from BaseConfigAdmin. It doesn't happen like that here. So, I think this code is necessary. Otherwise, the change and add views break for Node and Link. What say @nemesisdesign?